### PR TITLE
add binning with test.

### PR DESCRIPTION
**Pull request by MrAE**

### CODE REVIEW

File: Python/rerf/RerF.py [hunk 1]
```diff
@@ -52,6 +55,11 @@ def fastRerF(
     fractionOfFeaturesToTest : float, optional
         Sets mtry based on a fraction of the features instead of an 
         exact number (default: None).
+    binMin : int, optional
+        If set to greater than 0, this is used as the minimum number of
+        samples in the node for which to use subsampled points.
+    binSize : int, to be used with binMin
```
Comment (falkben) [inline Python/rerf/RerF.py:67]: is binSize the number of bins or the number of samples in each bin?

File: Python/rerf/RerF.py [hunk 2]
```diff
@@ -83,6 +91,14 @@ def fastRerF(
     forestClass.setParameter("numTreesInForest", trees)
     forestClass.setParameter("minParent", minParent)
 
+    if binMin > 0:
+        forestClass.setParameter("binMin", binMin)
+        forestClass.setParameter("binSize", binSize)
+        if not(forestType == "rerf" or forestType == "rfBase"):
```
Comment (falkben) [inline Python/rerf/RerF.py:115]: maybe move this above the set parameters?

**Review by falkben — CHANGES_REQUESTED** [review]

File: Python/rerf/RerF.py (hunk 1 above)
Comment (MrAE) [reply Python/rerf/RerF.py:67]: number of points in each bin. Confirmed from James.

File: Python/rerf/RerF.py (hunk 2 above)
Comment (falkben) [reply Python/rerf/RerF.py:115]: So, that's fine but I wonder if we should have the user facing parameter be number of bins? Basically which do we want to keep constant?

**Review by falkben — APPROVED** [review]
we should probably add this to sklearn bindings as well at some point.  making an issue.

**Review by falkben — CHANGES_REQUESTED** [review]
travis and I get similar error: pytest

`“pytest” terminated by signal SIGSEGV (Address boundary error)`

**Comment by falkben** [comment]
should this be closed for now?

**Pull request closed without merging**